vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -406,27 +416,27 @@ class MessageItemFactory @Inject constructor(
         }
 
         return MessageVoiceItem_()
```

Comment:
You should test the content scanner with a MessageVoiceItem
@giomfo I believe voice message is not supported on Tchap Android. But what happens if the app receives a voice message from another client ?

Reply:
voice message is not supported on Tchap FTM
I agree we should test the current behavior in case of incoming voice msg -> @Claire1817 can you please create a dedicated issue to check incoming voice message (displayed?/play?/antivirus?)